src/api.coffee

```diff
@@ -17,9 +17,15 @@ module.exports = (application) ->
 	parsedRouter.use(bodyParser())
 
 	api.use (req, res, next) ->
+		queryKey = req.query.apikey
+		header = req.get('Authorization')
+		match = header.match(/^ApiKey ([a-f\d]+)$/)
```

Comment:
This still won't work for 'bananas', those troublesome `n`s, not to mention that `s` :P